src/NHibernate/Tool/hbm2ddl/SchemaUpdate.cs

```diff
@@ -42,6 +43,24 @@ public SchemaUpdate(Configuration cfg, Settings settings)
 			formatter = (settings.SqlStatementLogger.FormatSql ? FormatStyle.Ddl : FormatStyle.None).Formatter;
 		}
 
+		private void Initialize()
+		{
+			if (wasInitialized)
+			{
+				return;
+			}
+
+			string autoKeyWordsImport = PropertiesHelper.GetString(Environment.Hbm2ddlKeyWords, configuration.Properties, "not-defined");
+			autoKeyWordsImport = autoKeyWordsImport.ToLowerInvariant();
+			if (autoKeyWordsImport == Hbm2DDLKeyWords.AutoQuote)
```

Comment:
> Here, probably, shall be the same check as in SessionFactoryImpl.

It would look more consistent, but would complicate code for nothing, in its current state. The check in `SessionFactoryImpl` treats `AutoQuote` and `KeywordImport` as two separated options. But they indeed are only one configuration setting, currently not allowing having auto-quoting without having keyword-import.

To have things actually consistent, we would need to put a helper somewhere for getting those two booleans from the actual configuration setting, or refactor `SchemaUpdate` and `SchemaExport` for taking/building a session factory setting object. I do not think it is worth the hassle.